server/chat-commands/room-settings.ts

```diff
@@ -1114,47 +1143,26 @@ export const commands: ChatCommands = {
 	],
 
 	officialchatroom: 'officialroom',
-	officialroom(target, room, user) {
-		room = this.requireRoom();
-		this.checkCan('makeroom');
-		if (!room.persist) {
-			return this.errorReply(`/officialroom - This room can't be made official`);
-		}
-		if (this.meansNo(target)) {
-			if (!room.settings.isOfficial) return this.errorReply(`This chat room is already unofficial.`);
-			delete room.settings.isOfficial;
-			this.addModAction(`${user.name} made this chat room unofficial.`);
-			this.modlog('UNOFFICIALROOM');
-			delete room.settings.isOfficial;
-			room.saveSettings();
-		} else {
-			if (room.settings.isOfficial) return this.errorReply(`This chat room is already official.`);
-			room.settings.isOfficial = true;
-			this.addModAction(`${user.name} made this chat room official.`);
-			this.modlog('OFFICIALROOM');
-			room.settings.isOfficial = true;
-			room.saveSettings();
-		}
+	officialroom() {
+		this.parse(`/setroomsection officialrooms`);
 	},
 
 	psplwinnerroom(target, room, user) {
 		this.checkCan('makeroom');
 		room = this.requireRoom();
 		if (!room.persist) {
-			return this.errorReply(`/psplwinnerroom - This room can't be marked as a PSPL Winner room`);
+			return this.errorReply(`/psplwinnerroom - This room can't be marked as a PSPL Winner room.`);
 		}
 		if (this.meansNo(target)) {
 			if (!room.settings.pspl) return this.errorReply(`This chat room is already not a PSPL Winner room.`);
-			delete room.settings.pspl;
 			this.addModAction(`${user.name} made this chat room no longer a PSPL Winner room.`);
-			this.modlog('PSPLROOM');
+			this.globalModlog('UNPSPLROOM');
```

Comment:
```suggestion
			this.globalModlog('UNPSPLROOM', null);
```

Reply:
null isn't required here - this has no effect.

Reply:
> null isn't required here - this has no effect.
null is for : Expected 2-4 arguments, but got 1

Reply:
Huh. That certainly shouldn't be required. It seems like the optimal solution is to mark the user parameter as optional, instead (as it used to be?)

Reply:
Yes. See d5a4c7d9e2a14ad83503eb5c455f9063fc8e60a8.